src/style.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+}
+
+.search {
```

Comment:
It would be better to avoid margin collapsing ([article](https://www.freecodecamp.org/news/what-is-margin-collapse-and-how-to-avoid-it/).) 
So in this case you can just use margin-top for both form elements